libs/platform/src/lib/table/components/table-p13-dialog/sorting/sorting.component.ts

```diff
@@ -132,6 +132,11 @@ export class P13SortingDialogComponent implements Resettable {
         return rule.columnKey;
     }
 
+    _getAriaLabel(rule: ValidatedSortRule): string {
+        const columnLabel = this.columns.find((col) => col.key === rule.columnKey)?.label;
```

Comment:
Can you calculate these values to a map on component init? Since this function is called from UI, it will run `find` on each CD cycle

Reply:
Done